src/Symfony/Component/HttpKernel/Tests/HttpCache/HttpCacheTest.php

```diff
@@ -1242,7 +1242,6 @@ public function testEsiCacheForceValidation()
         $this->assertEquals('Hello World! My name is Bobby.', $this->response->getContent());
         $this->assertNull($this->response->getTtl());
         $this->assertTrue($this->response->mustRevalidate());
-        $this->assertTrue($this->response->headers->hasCacheControlDirective('private'));
```

Comment:
Shouldn’t this be changed to `$this->assertFalse(…)` instead of removing the assertion?

Reply:
I don't think so, as this _tests that esi cache forces validation_, and that case is validated with the line above. The cache header is not related to esi cache validation.

Reply:
I guess the `Hello World` response is `private` by default, since is does not explicitly set `public` or a `s-maxage`? 

If so, shouldn’t `private` then be preserved?

Reply:
Or am I confusing behavior of the `Response` class with how this test works?